content/doc/administration/requirements/java.adoc

```diff
@@ -1,11 +1,11 @@
 ---
 layout: documentation
-title:  Java requirements
+title:  Java requirements(https://plugins.jenkins.io/jdk-tool/)
```

Comment:
```suggestion
title:  Java requirements(https://plugins.jenkins.io/jdk-tool/)
```
The title should not have a link included, as it will not render correctly. This is also out of sync with the rest of the Jenkins documentation (which do not have links in their page titles). [The asciidoc reference guide shows what can be done in the doc header.](https://docs.asciidoctor.org/asciidoc/latest/document/header/)

It would be great to have the link somewhere in the page where it has context or can be introduced. Having it linked in the title would not provide any sort of description or reasoning for it. Also, the plugin page has several instances that say it is **not** good for production environments. If this is still the case, it may not be best to recommend. If this no longer applies, the plugin documentation would need to be updated to reflect that.